src/styles/tailwind.css

```diff
@@ -116,14 +116,50 @@ body {
   .list-wrapper{
     @apply max-w-[382px] m-3 p-4 border-[1px] rounded-lg border-custom-burnt-orange bg-custom-cream
   }
+}
 
-  .task-wrapper{
-    @apply max-w-[300px] m-2 p-2 border-custom-yellow rounded-lg bg-custom-yellow
-  }
+/* mascot css code */
+.task-wrapper{
+  border-color: red;
+  border-width: 3px;
+  margin: 5px;
+}
+
+.button {
+  cursor: pointer;
+}
 
+.container {
+  text-align: center;
+  margin-top: 50px;
 }
 
-.Collapsible__trigger{
```

Comment:
This collapsible-trigger needs to be brought back. 

Just add a closing bracket at the end
`.Collapsible__trigger{
  color: #0E8992
}`